### PR TITLE
doc: update CHANGELOG for v1.37.0 release

### PR DESCRIPTION
**Pull request by devbww**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/8474)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -144,6 +146,32 @@ best long-term interest of our customers.
...
 or reason to be extending this class.
 
+**OTHER CHANGES**:
+
+* feat(bigtable): better support for PSC and VPC-SC ([#8458](https://github.com/googleapis/google-cloud-cpp/pull/8458))
+* refactor(bigtable)!: mark InstanceAdminClient as final ([#8449](https://github.com/googleapis/google-cloud-cpp/pull/8449))
+* fix(bigtable): use the appropriate type alias for AsyncRowReader::last_read_row_key_ ([#8437](https://github.com/googleapis/google-cloud-cpp/pull/8437))
+* docs(bigtable): cleanup admin documentation ([#8436](https://github.com/googleapis/google-cloud-cpp/pull/8436))
+* fix(bigtable): ReadRows retries from the `last_scanned_row_key` ([#8423](https://github.com/googleapis/google-cloud-cpp/pull/8423))
+* refactor(bigtable)!: modernize TableAdmin ([#8383](https://github.com/googleapis/google-cloud-cpp/pull/8383))
```
Comment (dbolduc) [inline CHANGELOG.md:156]: and I'd delete this entry. (it is detailed above as a breaking change)

File: CHANGELOG.md [hunk 2]
```diff
@@ -144,6 +146,32 @@ best long-term interest of our customers.
...
+
+* docs(iam): added comments on private_key_data ([#8204](https://github.com/googleapis/google-cloud-cpp/pull/8204))
+
+### [Pub/Sub](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsub/README.md)
+
+* feat(pubsub): support `AuthorityOption` ([#8460](https://github.com/googleapis/google-cloud-cpp/pull/8460))
+* feat(pubsub): support x-goog-user-project ([#8456](https://github.com/googleapis/google-cloud-cpp/pull/8456))
+* feat(pubsub): per-call Options for SubscriptionAdminClient ([#8414](https://github.com/googleapis/google-cloud-cpp/pull/8414))
+* feat(pubsub): per-call options in TopicAdminClient ([#8411](https://github.com/googleapis/google-cloud-cpp/pull/8411))
+* fix(pubsub): restore MakeSchemaAdminConnection overload ([#8412](https://github.com/googleapis/google-cloud-cpp/pull/8412))
```
Comment (dbolduc) [inline CHANGELOG.md:169]: I think we can delete this too. I think there was no change from release to release.

File: CHANGELOG.md [hunk 3]
```diff
@@ -144,6 +146,32 @@ best long-term interest of our customers.
...
+* feat(pubsub): per-call options in SchemaAdminClient ([#8406](https://github.com/googleapis/google-cloud-cpp/pull/8406))
+* doc(pubsub): add region tag for subscription with filter ([#8326](https://github.com/googleapis/google-cloud-cpp/pull/8326))
+* feat(pubsublite): resumable async streaming read write rpc ([#8233](https://github.com/googleapis/google-cloud-cpp/pull/8233))
+* feat(pubsublite): resumable async read write rpc stream boilerplate ([#8441](https://github.com/googleapis/google-cloud-cpp/pull/8441))
```
Comment (dbolduc) [inline CHANGELOG.md:173]: I think these are really `impl`s that we can delete.

File: CHANGELOG.md [hunk 4]
```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
...
-### New Libraries
+**OTHER CHANGES**:
+
+* feat(spanner): use prevailing Options in ConnectionImpl ([#8466](https://github.com/googleapis/google-cloud-cpp/pull/8466))
+* feat(spanner): convert ReadOptions/PartitionOptions to Options ([#8448](https://github.com/googleapis/google-cloud-cpp/pull/8448))
+* fix(spanner): instantiate OptionsSpan objects in legacy admin calls ([#8440](https://github.com/googleapis/google-cloud-cpp/pull/8440))
+* feat(spanner): convert QueryOptions to Options in main client API ([#8430](https://github.com/googleapis/google-cloud-cpp/pull/8430))
+* fix(spanner): add Options save/restore to PartialResultSetSource ([#8355](https://github.com/googleapis/google-cloud-cpp/pull/8355))
+* feat(spanner): implement x-goog-user-project, part 2 ([#8332](https://github.com/googleapis/google-cloud-cpp/pull/8332))
+* feat(spanner): support x-goog-user-project ([#8316](https://github.com/googleapis/google-cloud-cpp/pull/8316))
```
Comment (dbolduc) [inline CHANGELOG.md:192]: Maybe consolidate these into one entry?

```
* feat(spanner): support x-goog-user-project ([#8316](https://github.com/googleapis/google-cloud-cpp/pull/8316), [#8332](https://github.com/googleapis/google-cloud-cpp/pull/8332))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
...
+
+### [Storage](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/storage/README.md)
 
-<!-- TODO: update library count with next release -->
+* feat(storage): support AuthorityOption ([#8462](https://github.com/googleapis/google-cloud-cpp/pull/8462))
+* fix(storage): json["projectTeam"] might be present but null ([#8446](https://github.com/googleapis/google-cloud-cpp/pull/8446))
+* feat(GCS+gRPC): implement `SetBucketIamPolicy()` ([#8431](https://github.com/googleapis/google-cloud-cpp/pull/8431))
```
Comment (dbolduc) [inline CHANGELOG.md:198]: @coryan decided not to include GCS+gRPC stuff in the CHANGELOG. we can delete all of them.

Previous discussion: https://github.com/googleapis/google-cloud-cpp/pull/8208#discussion_r796932188

File: CHANGELOG.md [hunk 6]
```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
...
+* feat(GCS+gRPC): implement `CreateBucket()` ([#8372](https://github.com/googleapis/google-cloud-cpp/pull/8372))
+* doc(storage): show how to configure endpoints ([#8354](https://github.com/googleapis/google-cloud-cpp/pull/8354))
+* feat(GCS+gRPC): implement `UpdateObject()` ([#8278](https://github.com/googleapis/google-cloud-cpp/pull/8278))
+* feat(GCS+gRPC): implement `ComposeObject()` ([#8209](https://github.com/googleapis/google-cloud-cpp/pull/8209))
+* doc(storage): UserIp option is deprecated ([#8468](https://github.com/googleapis/google-cloud-cpp/pull/8468))
+
+### [Common Libraries](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/README.md)
+
+* feat(generator): all bidir streams are experimental ([#8471](https://github.com/googleapis/google-cloud-cpp/pull/8471))
+* doc: update GA library list  ([#8467](https://github.com/googleapis/google-cloud-cpp/pull/8467))
```
Comment (dbolduc) [inline CHANGELOG.md:218]: the library list should be in this file, so we can delete this entry.

File: CHANGELOG.md [hunk 7]
```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
...
+* feat: better support for PSC and VPC-SC ([#8453](https://github.com/googleapis/google-cloud-cpp/pull/8453))
+* fix: wait until AutomaticallyCreatedBackgroundThreads start ([#8452](https://github.com/googleapis/google-cloud-cpp/pull/8452))
+* fix(oauth2): compute engine credential query ([#8421](https://github.com/googleapis/google-cloud-cpp/pull/8421))
```
Comment (dbolduc) [inline CHANGELOG.md:221]: internal. can delete.

File: CHANGELOG.md [hunk 8]
```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
...
+* fix: restore Options over deletion of StreamRange<T>::reader_ ([#8403](https://github.com/googleapis/google-cloud-cpp/pull/8403))
+* feat(common): ddd converting constructors for future ([#8329](https://github.com/googleapis/google-cloud-cpp/pull/8329))
+* fix(rest): formatting headers ([#8391](https://github.com/googleapis/google-cloud-cpp/pull/8391))
```
Comment (dbolduc) [inline CHANGELOG.md:224]: internal. can delete.

File: CHANGELOG.md [hunk 9]
```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
...
+* fix: add Options save/restore to StreamRange<T> and cancellations ([#8256](https://github.com/googleapis/google-cloud-cpp/pull/8256))
+* feat(generator): support parameters named "options" ([#8283](https://github.com/googleapis/google-cloud-cpp/pull/8283))
+* feat(generator): support `x-goog-user-project` ([#8245](https://github.com/googleapis/google-cloud-cpp/pull/8245))
+* fix(rest): correct detection of all headers received ([#8268](https://github.com/googleapis/google-cloud-cpp/pull/8268))
```
Comment (dbolduc) [inline CHANGELOG.md:228]: internal. can delete.

File: CHANGELOG.md [hunk 10]
```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
...
+* fix: correct uses of `target_compatible_with` ([#8257](https://github.com/googleapis/google-cloud-cpp/pull/8257))
+* fix(oauth2): use correct expiration slack value ([#8255](https://github.com/googleapis/google-cloud-cpp/pull/8255))
```
Comment (dbolduc) [inline CHANGELOG.md:230]: internal. can delete.

**Review by dbolduc — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 11]
```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
...
+
+### [Common Libraries](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/README.md)
+
+* feat(generator): all bidir streams are experimental ([#8471](https://github.com/googleapis/google-cloud-cpp/pull/8471))
+* doc: update GA library list  ([#8467](https://github.com/googleapis/google-cloud-cpp/pull/8467))
+* feat: better support for PSC and VPC-SC ([#8453](https://github.com/googleapis/google-cloud-cpp/pull/8453))
+* fix: wait until AutomaticallyCreatedBackgroundThreads start ([#8452](https://github.com/googleapis/google-cloud-cpp/pull/8452))
+* fix(oauth2): compute engine credential query ([#8421](https://github.com/googleapis/google-cloud-cpp/pull/8421))
+* fix: restore Options over deletion of StreamRange<T>::reader_ ([#8403](https://github.com/googleapis/google-cloud-cpp/pull/8403))
+* feat(common): ddd converting constructors for future ([#8329](https://github.com/googleapis/google-cloud-cpp/pull/8329))
```
Comment (coryan) [inline CHANGELOG.md:223]: I think `ddd` is a typo, just remove please.

**Review by coryan — COMMENTED** [review]
Most what dbolduc@ said.

File: CHANGELOG.md [hunk 12]
```diff
@@ -144,6 +146,32 @@ best long-term interest of our customers.
 as `final`. After the changes in [v1.36.0](#v1360---2022-02), there is no need
 or reason to be extending this class.
 
+**OTHER CHANGES**:
+
+* feat(bigtable): better support for PSC and VPC-SC ([#8458](https://github.com/googleapis/google-cloud-cpp/pull/8458))
+* refactor(bigtable)!: mark InstanceAdminClient as final ([#8449](https://github.com/googleapis/google-cloud-cpp/pull/8449))
+* fix(bigtable): use the appropriate type alias for AsyncRowReader::last_read_row_key_ ([#8437](https://github.com/googleapis/google-cloud-cpp/pull/8437))
+* docs(bigtable): cleanup admin documentation ([#8436](https://github.com/googleapis/google-cloud-cpp/pull/8436))
```
[the remark this answers is not in the document]
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 4 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 5 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 6 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 7 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 8 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 11 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

**Review by devbww — COMMENTED** [review]
Thanks.

File: CHANGELOG.md [hunk 13]
```diff
@@ -188,46 +181,27 @@ deprecated as they are still used in the `spanner::Connection` interface.)
...
 
 ### [Common Libraries](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/README.md)
 
 * feat(generator): all bidir streams are experimental ([#8471](https://github.com/googleapis/google-cloud-cpp/pull/8471))
-* doc: update GA library list  ([#8467](https://github.com/googleapis/google-cloud-cpp/pull/8467))
 * feat: better support for PSC and VPC-SC ([#8453](https://github.com/googleapis/google-cloud-cpp/pull/8453))
 * fix: wait until AutomaticallyCreatedBackgroundThreads start ([#8452](https://github.com/googleapis/google-cloud-cpp/pull/8452))
-* fix(oauth2): compute engine credential query ([#8421](https://github.com/googleapis/google-cloud-cpp/pull/8421))
 * fix: restore Options over deletion of StreamRange<T>::reader_ ([#8403](https://github.com/googleapis/google-cloud-cpp/pull/8403))
-* feat(common): ddd converting constructors for future ([#8329](https://github.com/googleapis/google-cloud-cpp/pull/8329))
```
Comment (dbolduc) [inline CHANGELOG.md]: I think coryan meant `s/ddd//`. Not to delete the whole entry.

File: CHANGELOG.md [hunk 14]
```diff
@@ -188,46 +181,27 @@ deprecated as they are still used in the `spanner::Connection` interface.)
 * fix(spanner): instantiate OptionsSpan objects in legacy admin calls ([#8440](https://github.com/googleapis/google-cloud-cpp/pull/8440))
 * feat(spanner): convert QueryOptions to Options in main client API ([#8430](https://github.com/googleapis/google-cloud-cpp/pull/8430))
 * fix(spanner): add Options save/restore to PartialResultSetSource ([#8355](https://github.com/googleapis/google-cloud-cpp/pull/8355))
-* feat(spanner): implement x-goog-user-project, part 2 ([#8332](https://github.com/googleapis/google-cloud-cpp/pull/8332))
 * feat(spanner): support x-goog-user-project ([#8316](https://github.com/googleapis/google-cloud-cpp/pull/8316))
```
Comment (dbolduc) [inline CHANGELOG.md:184]: did you want to also link to #8332 ?

File: CHANGELOG.md (hunk 13 above)
Comment (devbww) [reply CHANGELOG.md]: And so restored (although editing commit messages at this stage seems more confusing to me).

**Review by dbolduc — APPROVED** [review]

File: CHANGELOG.md (hunk 14 above)
Comment (devbww) [reply CHANGELOG.md:184]: I would proffer that we should either not mention it at all, or list it as a separate commit.

**Review by coryan — APPROVED** [review]

**Pull request merged**